### ISSUE TITLE
Inspec found not available after `gem install inspec`

### ISSUE THREAD

**Issue by darkn3rd** [inspec/inspec#4138]
### Problem

After `gem install inspec` there is no executable binary for InSpec.

### Steps

```bash
#### RUBY INSTALL
curl -fsSL https://github.com/rbenv/rbenv-installer/raw/master/bin/rbenv-installer | bash
echo 'export PATH="$HOME/.rbenv/bin:$PATH"' >> ~/.bashrc
echo 'eval "$(rbenv init -)"' >> ~/.bashrc
export PATH="$HOME/.rbenv/bin:$PATH"
eval "$(rbenv init -)"
rbenv install 2.5.3
rbenv global 2.5.3

#### INSPEC INSTALL
gem install inspec
rbenv rehash
```

### ACTUAL RESULTS
```
-bash: inspec: command not found
```

This also happens on macOS as well, using rbenv installed ruby.  Reproduces in clean Ubuntu 16.04 environment.

### EXPECTED RESULTS

Inspec would be somewhere in the path.  

### NOTES

```bash
gem list | grep inspec
inspec (4.3.2)

gem env
RubyGems Environment:
  - RUBYGEMS VERSION: 2.7.6
  - RUBY VERSION: 2.5.3 (2018-10-18 patchlevel 105) [x86_64-linux]
  - INSTALLATION DIRECTORY: /home/vagrant/.rbenv/versions/2.5.3/lib/ruby/gems/2.5.0
  - USER INSTALLATION DIRECTORY: /home/vagrant/.gem/ruby/2.5.0
  - RUBY EXECUTABLE: /home/vagrant/.rbenv/versions/2.5.3/bin/ruby
  - EXECUTABLE DIRECTORY: /home/vagrant/.rbenv/versions/2.5.3/bin
  - SPEC CACHE DIRECTORY: /home/vagrant/.gem/specs
  - SYSTEM CONFIGURATION DIRECTORY: /home/vagrant/.rbenv/versions/2.5.3/etc
  - RUBYGEMS PLATFORMS:
    - ruby
    - x86_64-linux
  - GEM PATHS:
     - /home/vagrant/.rbenv/versions/2.5.3/lib/ruby/gems/2.5.0
     - /home/vagrant/.gem/ruby/2.5.0
  - GEM CONFIGURATION:
     - :update_sources => true
     - :verbose => true
     - :backtrace => false
     - :bulk_threshold => 1000
     - "gem" => "--no-document"
  - REMOTE SOURCES:
     - https://rubygems.org/
  - SHELL PATH:
     - /home/vagrant/.rbenv/versions/2.5.3/bin
     - /home/vagrant/.rbenv/libexec
     - /home/vagrant/.rbenv/plugins/ruby-build/bin
     - /home/vagrant/bin
     - /home/vagrant/.local/bin
     - /home/vagrant/.rbenv/shims
     - /home/vagrant/.rbenv/bin
     - /home/vagrant/.pyenv/plugins/pyenv-virtualenv/shims
     - /home/vagrant/.pyenv/shims
     - /home/vagrant/.pyenv/bin
     - /usr/local/sbin
     - /usr/local/bin
     - /usr/sbin
     - /usr/bin
     - /sbin
     - /bin
     - /usr/games
     - /usr/local/games
     - /snap/bin
```

**Comment by tbugfinder** [comment]
just learned that the later version requires an additional "gem inspec-bin".

**Comment by darkn3rd** [comment]
I did after the fact, one community member not employed by Chef was thoughtful to explain that the change was due to licensing scheme, where executables require a license agreement.  This will confuse a lot of people unfortunately and cause a lot of disruption.

Anyhow, I guess this bug can be an enhancement request for docs.

**Comment by clintoncwolfe** [comment]
Thanks for the excellent report; we've just pushed the docs update. Thanks for your patience.

Dupe of 4145 fixed in #4163
